ql/src/java/org/apache/hadoop/hive/ql/exec/repl/OptimisedBootstrapUtils.java

```diff
@@ -70,6 +71,8 @@
   /** event ack file which contains the event id till which the cluster was last loaded. */
   public static final String EVENT_ACK_FILE = "event_ack";
 
+  public static final String BOOTSTRAP_TABLES_LIST = "bootstrap_table_list";
```

Comment:
Can we rename this file as _failover_bootstrap_table_list

Reply:
What would happen in rollback case , like we initiated the failover but aborted the process in between. Theoretically, upto what point will we allow that to happen?

Reply:
Renamed.

Answered for the rollback & failback above, on high level it is part of reset solution, and we allow that at all levels and stages using that. Details available in the doc